src/components/CommentCard.jsx

```diff
@@ -1,20 +1,41 @@
 import Card from "react-bootstrap/Card";
 import { useState } from "react";
 import axios from "axios";
-import { useParams } from "react-router-dom";
 
 export default function ({ comment }) {
   const [votes, setVotes] = useState(0);
   const [buttonClicked, setButtonClicked] = useState(false);
-  const { article_id } = useParams();
+  const [isDeleting, setIsDeleting] = useState(false);
+  const [deleted, setDeleted] = useState(false);
 
   const handleClick = (event) => {
     setVotes((currentVotes) => currentVotes + 1);
     setButtonClicked(true);
   };
 
+  const handleDelete = (event) => {
+    setIsDeleting(true);
+    axios.delete(`https://nc-news-53nl.onrender.com/api/comments/${comment.comment_id}`)
+    .then((response) => {
+      setDeleted(true);
+      setIsDeleting(false);
+    })
+    .catch((error) => {
+      console.log(error, '<---error in delete comment')
+      setDeleted(false);
+      setIsDeleting(false);
+    })
+  }
+
+  if (deleted) {
+    return <p>Your comment has been deleted.</p>
+  }
+
+  const user = 'grumpy19';
```

Comment:
It is fine to hard code this - but I would let the user know who they are logged in as (unless you plan to do this later with a log in)